editor/src/messages/portfolio/document_migration.rs

```diff
@@ -1357,51 +1279,8 @@ fn migrate_node(node_id: &NodeId, node: &DocumentNode, network_path: &[NodeId],
 		document.network_interface.set_input(&InputConnector::node(*node_id, 2), old_inputs[3].clone(), network_path);
 	}
 
-	if reference == "Flatten Vector Elements" {
```

Comment:
not possible to include this migration since it relies on changing reference to not run again, which is no longer possible.